plan/dag_plan_test.go

```diff
@@ -489,6 +492,9 @@ func (s *testPlanSuite) TestDAGPlanBuilderBasePhysicalPlan(c *C) {
 	se, err := tidb.CreateSession(store)
 	c.Assert(err, IsNil)
 
+	_, err = se.Execute("use test")
```

Comment:
Why add this?

Reply:
Because `show tables` will cause an `No DB found` error
Before this commit, we resolve `ShowStmt` in `resolver.go` which is invoked with a `db parameter`  in `plan.MockResolve()`.
After this commit, we move the resolver to preprocessor.go with `no input parameter for dbname`,
so we have to `use test` manually.